integrations/claude_query_server.py

```
@@ -0,0 +1,131 @@
+)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--port", type=int, default=SERVER_PORT)
+    args = parser.parse_args()
+    print(f"AI Tools Knowledge Base server — http://localhost:{args.port}")
+    print(f"  Tools:   GET  http://localhost:{args.port}/tools")
+    print(f"  A2A:     GET  http://localhost:{args.port}/.well-known/agent.json")
+    print(f"  Query:   POST http://localhost:{args.port}/query")
+    uvicorn.run(app, host="0.0.0.0", port=args.port, log_level="info")
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)